### ISSUE TITLE
Fix loging to chat

### ISSUE THREAD

**Issue by bukson** [bukson/steampy#128]

**Comment by bukson** [comment]
As it seems it now uses the websocets to communicate. 

I don't know if the old chat api is accessable in some way, if anyone knows something please share, otherwise I think that I should drop the chat module and go to version 0.6

**Comment by MeridianPY** [comment]
oh. so this means the chat module isnt working anymore?

**Comment by bukson** [comment]
Yes, unfortunetally it is now working on websocets and I have little experience in reverse engineering this protocol

**Comment by MeridianPY** [comment]
I would be willing to help but since I just started learning Python and have no clue of API's whatsoever I don't think that would help :/

**Comment by Gobot1234** [comment]
I'd be willing to help as I'm currently looking at getting it working :)
